### ISSUE TITLE
Validation errors JSON data copied to clipboard lacks sources

### ISSUE THREAD

**Issue by westonruter** [Automattic/amp-wp#5825]
## Bug Description

This appears to be something that was missed in #5500 for #5209.

I just noticed when doing the Copy to Clipboard function:

![image](https://user-images.githubusercontent.com/134745/106501258-c5f53780-6477-11eb-8519-fa52b3c1fa2e.png)

The result does not include `sources`:

```json
[
    {
        "code": "DISALLOWED_TAG",
        "node_attributes": {
            "src": "https://wordpressdev.lndo.site/content/plugins/bad-block/front.js?ver=__normalized__",
            "id": "bad-block-js"
        },
        "node_name": "script",
        "node_type": "ELEMENT",
        "parent_name": "head",
        "type": "js_error",
        "removed": true,
        "reviewed": false
    },
    {
        "code": "DISALLOWED_TAG",
        "node_attributes": [],
        "node_name": "script",
        "node_type": "ELEMENT",
        "parent_name": "span",
        "text": "document.write(\"Bad dynamic block!!!\");",
        "type": "js_error",
        "removed": true,
        "reviewed": false
    },
    {
        "code": "DISALLOWED_TAG",
        "node_attributes": [],
        "node_name": "bad",
        "node_type": "ELEMENT",
        "parent_name": "div",
        "type": "html_element_error",
        "removed": true,
        "reviewed": false
    },
    {
        "code": "DISALLOWED_TAG",
        "node_attributes": [],
        "node_name": "ok",
        "node_type": "ELEMENT",
        "parent_name": "div",
        "type": "html_element_error",
        "removed": true,
        "reviewed": false
    }
]
```

## Expected Behaviour

The array of source(s) should be included in the errors data.

---------------

_Do not alter or remove anything below. The following sections will be managed by moderators only._

## Acceptance criteria

* 

## Implementation brief

* 

## QA testing instructions

* 

## Demo

* 

## Changelog entry

*

**Comment by westonruter** [comment]
I thought this feature had been merged into the 2.0 branch, but no, it's milestoned for 2.1. So re-milestoning the bug for 2.1 as well.

**Comment by westonruter** [comment]
**QA passed**

Sources included in JSON as seen in https://github.com/ampproject/amp-wp/issues/5209#issuecomment-826249912.
